apigen.neon

```diff
@@ -1,6 +1,6 @@
 source: ./
 
-destination: docs.woothemes.com/images/sensei-apidocs
+destination: docs.woocommerce.com/images/sensei-apidocs
```

Comment:
`docs.woocommerce.com/images/sensei-apidocs` doesn't exist. Should we change this to `docs.woocommerce.com/sensei-apidocs`?